tests/integ/test_function.py

```diff
@@ -1072,25 +1072,31 @@ def test_as_negative(session):
 
 
 @pytest.mark.localtest
-def test_to_date_to_array_to_variant_to_object(session):
+def test_to_date_to_array_to_variant_to_object(session, local_testing_mode):
     df = (
-        session.create_dataframe([["2013-05-17", 1, 3.14, '{"a":1}']])
+        session.create_dataframe(
+            [["2013-05-17", 1, 3.14, '{"a":1}'], [None, None, None, None]]
+        )
         .to_df("date", "array", "var", "obj")
         .with_column("json", parse_json("obj"))
     )
 
     df1 = df.select(
         to_date("date"), to_array("array"), to_variant("var"), to_object("json")
     )
-    df2 = df.select(
-        to_date(col("date")),
-        to_array(col("array")),
-        to_variant(col("var")),
-        to_object(col("json")),
-    )
 
-    res1, res2 = df1.collect(), df2.collect()
-    assert res1 == res2
```

Comment:
may I ask why this is removed? looks like originally it is testing `col` function call.